packages/tables/src/Columns/Summarizers/Count.php

```diff
@@ -37,7 +37,7 @@ public function summarize(Builder $query, string $attribute): int | float | arra
 
         $state = [];
 
-        foreach ($query->clone()->pluck($attribute) as $value) {
+        foreach ($query->clone()->pluck($attribute)->uniqueStrict() as $value) {
```

Comment:
Maybe this would work better instead, on the database level? I haven't tested it, can you please?

```suggestion
        foreach ($query->clone()->distinct()->pluck($attribute) as $value) {
```

Reply:
that's the good idea, its work as expected